components/basecomponents/SEO.tsx

```diff
@@ -3,25 +3,26 @@ import Head from 'next/head';
 
 interface Props {
   title: string;
-  description: string;
-  image: string;
+  description?: string;
+  image?: string;
+  noreferrer?: boolean;
 }
 
-const SEO = ({ title, description, image }: Props) => {
+const SEO = ({ title, description, image = 'https://www.movapp.cz/icons/movapp-cover.jpg', noreferrer = true }: Props) => {
```

Comment:
You can also see all uses of `SEO` and remove the `image` prop if it contains the default value :)